reference/snapshot.md

```diff
@@ -0,0 +1,279 @@
+---
+Title: OpenEBS Volume Snapshot
+Sidebar: Volume snapshot
+---
+
+**Volume snapshots** are copies of a persistent volume claims (PVCs) at a specific point in time. They can be used to restore a volume to a previous state or create a new volume. Mayastor provides support for industry standard copy-on-write (COW) snapshots, which is a po.pular methodology for taking snapshots by keep track of only those blocks that have changed
```

Comment:
Minor change:  popular  instead of "po.pular".
Also "by keep track of"  could be "by keeping track of"